packages/react-error-overlay/src/index.js

```diff
@@ -18,22 +18,36 @@ import { iframeStyle, overlayStyle } from './styles';
 import { applyStyles } from './utils/dom/css';
 
 import type { ErrorRecord } from './listenToRuntimeErrors';
+import type { ErrorLocation } from './utils/parseCompileError';
 
 type RuntimeReportingOptions = {|
   onError: () => void,
-  launchEditorEndpoint: string,
   filename?: string,
 |};
 
+type OpenInEditorListener = (errorLoc: ErrorLocation) => void;
+
 let iframe: null | HTMLIFrameElement = null;
 let isLoadingIframe: boolean = false;
 
 let renderedElement: null | Element<any> = null;
+
+let openInEditorListener: null | OpenInEditorListener = null;
 let currentBuildError: null | string = null;
 let currentRuntimeErrorRecords: Array<ErrorRecord> = [];
 let currentRuntimeErrorOptions: null | RuntimeReportingOptions = null;
 let stopListeningToRuntimeErrors: null | (() => void) = null;
 
+export function listenToOpenInEditor(listener: OpenInEditorListener) {
```

Comment:
If this is renamed to `setEditorCallback`, we could make this accept `OpenInEditorListener | null` so it can be switched off by sending `null`.